tests/conftest.py

```diff
@@ -25,6 +29,15 @@ def no_local_kubeconfig():
     ):
         yield
 
+@pytest.fixture(autouse=True)
+def fork_multiprocess():
```

Comment:
The issue is actually with how we mock functions. Since we are patching objects directly the `patches` are only applied if we fork and not if we `spawn` since the original functions are re-imported. This means when we try to send a mocked function over the wire pickle can't find it during deserialization.